app/views/employment/AmountOfTipsOnP60View.scala.html

```diff
@@ -0,0 +1,74 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.headerFooterTemplate.Layout
+@import views.html.templates.helpers.{Button, ErrorSummary, Heading, InputRadio, InputText}
+@import utils.ViewUtils.bigDecimalCurrency
+
+@this(
+   layout: Layout,
+   formWithCSRF: FormWithCSRF,
+   errorSummary: ErrorSummary,
+   button: Button,
+   heading: Heading,
+   inputRadio: InputRadio,
+   inputText: InputText
+)
+@(form: Form[BigDecimal], taxYear: Int, employmentId:String, amount: Option[BigDecimal])(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+
+@hasFormErrors = @{form.hasErrors}
+
+@headingForUse = @{
+messages(s"amountsNotOnYourP60.heading.${if(user.isAgent) "agent" else "individual"}")
+}
+
+@layout(pageTitle = headingForUse, taxYear = Some(taxYear), hasErrors = hasFormErrors, isAgent = user.isAgent) {
+
+    @errorSummary(form.errors.distinct)
+
+    @formWithCSRF(action = controllers.employment.routes.AmountOfTipsOnP60Controller.submit(taxYear, employmentId)) {
+
+
+            @inputText(
+                form,
+                id = "amount",
+                name = "amount",
+                heading = headingForUse,
+                headingClasses = "govuk-! -margin-bottom-2",
+                subheading = Some(messages("employment.caption", (taxYear - 1).toString, taxYear.toString)),
+                content = Some(contentHtml),
+                hint = Some(Html(messages("common.currency.hint"))),
+                isPageHeading = false,
+                currency = true
+            )
+
+
+
+        @button()
+    }
+}
+
+@contentHtml = {
+    @if(amount != None){
```

Comment:
Option has a method `.isDefined` you could use here

Reply:
updated